docs/contributing.rst

```diff
@@ -83,6 +83,16 @@ Running Tests
 Linting
 -------
 
+This project uses [pre-commit](https://pre-commit.com/) for linting.
+
+1. install pre-commit hooks (this only needs to be done once)
+
+::
+
+   poetry run pre-commit install
+
+2. run/apply lints
+
```

Comment:
Not so happy with the docs.
I think pre-commit gets not introduced to support linting.
It's more or less a general quality gate which needs to be passed before a commit can be pushed.

So we need paragraph how to create and send PRs to this project.
If you like, you can already add it.
Or I will do it with a new PR (after yours got merged).

Reply:
can you suggest a wording that you prefer?

or i can leave it to you to update in a following commit? whichever you prefer